tests/utils/test-utils.tsx

```
@@ -0,0 +1,179 @@
+  
+
+/**
+ * Expects the auth status to be called with the given userID
+ * @param status - The status to expect
+ * @param userID - The userID to expect
+ */
+export const expectAuthStatus = (status: string, userID: string) => {
+    var checkStatus = null 
+    switch (status) {
+        case 'profile':
+            checkStatus = authUtils.checkProfileStatus(userID)
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)